sites/funmooc/src/backend/funmooc/settings.py

```diff
@@ -553,7 +556,10 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
                     environ_prefix=None,
                 ),
                 "OPTIONS": values.DictValue(
-                    {"CLIENT_CLASS": "richie.apps.core.cache.SentinelClient"},
+                    {
+                        "CLIENT_CLASS": "richie.apps.core.cache.SentinelClient",
+                        "FALLBACK_CLASS": "django.core.cache.backends.locmem.LocMemCache",
```

Comment:
the setting is called `BACKEND` and not `BACKEND_CLASS`, so maybe call it `FALLBACK_BACKEND` to differentiate from the the sentinel client class:
```suggestion
                        "FALLBACK_BACKEND": "django.core.cache.backends.locmem.LocMemCache",
```